src/sentry/models/grouphash.py

```diff
@@ -24,6 +24,7 @@ class State:
     project = FlexibleForeignKey('sentry.Project', null=True)
     hash = models.CharField(max_length=32)
     group = FlexibleForeignKey('sentry.Group', null=True)
+    group_tombstone = FlexibleForeignKey('sentry.GroupTombstone', null=True)
```

Comment:
🤔 

How important is the actual foreign key constraint here? From my quick glances, it seems it'd be fine to just use a `group_tombstone_id` without the constraint.

I don't think a blocker though since we could drop the constraint later if we needed to, for example, shard off this table, which we'd have to do for the other columns here too.

Also, this migration _will_ cause a very short hiccup I'm almost positive, so we should coordinate this deploy so we can preemptively stop workers and reduce pressure.